.github/workflows/check-semver.yml

```diff
@@ -136,3 +137,17 @@ jobs:
 
             exit 1
           fi
+          
+          # Only enforce SemVer restrictions for backports targeting stable branches
+          if [[ "$BASE_BRANCH" == stable* ]]; then
+            echo "🔍 Backport branch detected, checking for non-patch/none changes..."
+            
+            if grep -E 'bump:' prdoc/pr_$PR.prdoc | grep -Ev 'bump: (patch|none)' > /dev/null; then
```

Comment:
Raymond correctly pointed out that this check needs one more condition to let it pass when `validate: false` regardless of `bump` level.

Reply:
haven't looked at the code changes in detail, but I believe we can simplify by relying on the existing `semver` check.

The existing `semver` CI check is `required` for stable branches so it guarantees that either:
- bumps are correct,
  or
- `validate: false` was explicitly set to override certain bumps.

As such, this new check would have the overall desired result with the following simple rule:

IF `bump == major` AND `validate != false` => FAIL CI
ELSE => OK